tools/ldb_cmd.cc

```diff
@@ -2948,8 +3072,23 @@ void DumpWalFile(Options options, std::string wal_file, bool print_header,
             break;
           }
         }
-        row << WriteBatchInternal::Sequence(&batch) << ",";
-        row << WriteBatchInternal::Count(&batch) << ",";
+        SequenceNumber sequence_number = WriteBatchInternal::Sequence(&batch);
+        uint32_t batch_count = WriteBatchInternal::Count(&batch);
+        assert(prev_batch_seqno);
+        assert(prev_batch_count);
```

Comment:
We could probably also assert that either both have a value, or neither one does